docs/update-prod-data.md

```diff
@@ -0,0 +1,82 @@
+# Steps to update the dev db with prod data
+
+
+## 1 - Take a Dump of Data from Production and Export to a File
+
+You'll first need to take a dump of the data from the production environment and export it to a file. This file will contain all of the data that is currently in the production database.
+
+You will first have to use **kubectl** to get the list of available pods `kubectl get pod`
+Locate the one with the prefix `lists-postgres- `
+
+The you'll have to access the shell of that pod with the following command:
+```bash
+kubectl exec -it lists-postgres-[pod_id]
+```
+
+_Replace [pod_id] with the reelevant id, eg `764c9b5879-5mf5f`_
+
+Now you can run the following command:
+
+```bash
+pg_dump -U master -h lists-postgres:5432 -Fc lists > prod_dump.dump
+```
+
+## 2 - Load the Dump into a Local Development Database
+
+Once you have the dump file, you'll need to load it into a local development database so you can make changes without affecting the production environment.
+
+Here's an example of how to load a Postgres dump file into a local development database:
+
+```bash
+psql -U master -h localhost -d lists < prod_data.sql
+```
+
+
+## 3 - Update Emails on All Lists in Local Dev Database
+
+Now that you have the data from the production database loaded into your local development database, you can make the necessary changes. In this case, you need to update the emails on all of the lists in the database.
+
+The exact Postgres command to do this will depend on the schema of your database, but it might look something like this:
+
+```sql
+BEGIN;
+UPDATE List
+SET jsonData = jsonb_set(jsonData,'{users}', jsonb_build_array('list-management@cautionyourblast.com'))
+WHERE ("jsonData"->'users')::jsonb is not null;
+COMMIT;
+
+```
+
+### 3.a If dev data contains Ali's email
+
+If the data contains an email in the user's part of the jsonData column that resembles **ali.salaman@fcdo.gov.uk** you can use the following script to replace it with the propper email:
+
+```sql
+BEGIN;
+UPDATE List
+SET jsonData = jsonb_set(jsonData,'{users}',  (jsondata->'users')::jsonb || '["list-management@cautionyourblast.com"]'::jsonb)
+WHERE (jsonData->'users')::jsonb  ? 'ali.salaman@fcdo.gov.uk';
+COMMIT;
+
+BEGIN;
+UPDATE List
+SET jsonData = jsonb_set(jsonData,'{users}',(jsonData->'users') - 'ali.salaman@fcdo.gov.uk')
+WHERE (jsonData->'users')::jsonb  ? 'ali.salaman@fcdo.gov.uk';
+COMMIT;
+```
+
+## 4 - Take a Dump of Data from Local Dev Database
+
+After you've made your changes, you'll want to take another dump of the data from the local development database to create a new dump file that includes your changes.
```

Comment:
These are instructions -- not suggestions. Remove all instances of "you'll want". The reader _has_ to do these things (not want to do them).

Words like "after" and "finally" are filler as well. We know they are before or after because the steps are numbered.

Reply:
Wouldn't you say the tone of the documentation is subjective? Regardless of if I say `has` or `you'll want` the user still gets the correct information they need.

Reply:
On subjectivity -- no, not really, sorry..! Being a technical writer is a job - technical writing in general is a fairly well defined skill and tone. 

For documentation and more generally instructions, the grammatical mood (or verb mood) is mostly imperative. Sometimes a conditional mood is use "if you need X, do Y". 

But here with "you'll want", it's the subjunctive mood. Here it's the _writer's_ "wish" that you do something. Compared to the conditional mood, which is the _reader's_ wish to do something + an instruction (imperative).

This wasn't about replacing you'll want with has. 

I need/have to update the dev database (because it is my task). The information I get out is the same, that's true, but I'd prefer to get to the information quickly. "you'll want to" - is that a suggestion? _do_ I want to? what if I don't want to? (tough 💩 - you NEED to do it to get to the end result..!)


>After you've made your changes, you'll want to take another dump of the data from the local development database to create a new dump file that includes your changes.

can just be 
> Generate a new data dump from your local development database. 

This just applies to technical documentation. Things like blogs, podcasts etc isn't technical documentation. You can use whatever mood you like - and they are subjective.

Reply:
Fair point, I see where you're coming from 👍🏿